src/select/content/_select-content.scss

```
@@ -0,0 +1,25 @@
+.select-content {
+  position: absolute;
+
+  z-index: 1;
+
+  width: max-content;
+  min-width: 100%;
+
+  margin-top: 15px;
+  padding: 6px 0;
+
+  background-color: white;
+
+  /* stylelint-disable-next-line color-no-hex */
+  border: 1px solid #dfe3ef;
+  border-radius: 6px;
+
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05);
+
+  visibility: hidden;
+
+  &--is-visible {
+    visibility: visible;
+  }

```

How about using opacity with a transition instead? So that, we can change the transition on the project side.

```suggestion
  opacity: 0;
  
  transition: opacity 0.25s ease;

  &--is-visible {
    opacity: 1;
  }

```